corehq/pillows/case.py

```diff
@@ -102,10 +101,12 @@ def get_case_pillow(
         expected_topics = set(CASE_TOPICS + (DEMO_CASE_SQL,))  # the demo case sql topic is only used here
         assert set(topics).issubset(expected_topics), "This is a pillow to process cases only"
     else:
-        if include_demo_topic:
-            topics = CASE_TOPICS + (DEMO_CASE_SQL,)
-        else:
+        # if running a demo case pillow, no need for the case pillow process to consume changes from the demo topic
+        if settings.RUN_DEMO_CASE_PILLOW:
             topics = CASE_TOPICS
+        else:
+            # drain changes from the demo case sql topic
+            topics = CASE_TOPICS + (DEMO_CASE_SQL,)
```

Comment:
Didn't catch this before, but will this require that `DEMO_CASE_SQL` must exist on all environments? It seems ideal to only add it where it is needed, and not to have to worry about the additional complexity anywhere else.

Secondly, is it important that both topics have the same number of partitions?